rust/parser/typeql.pest

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+eof_query = { SOI ~ query ~ EOI }
+eof_queries = { SOI ~ query+ ~ EOI }
```

Comment:
will this accept multiple queries split over multiple lines if you don't do a `(query ~)+`?

Reply:
Yes, all repetition operators, i.e. `*`, `+`, `{m,n}` also allow implicit whitespace.

Reply:
good to know!